lib5k/src/main/java/io/github/frc5024/purepursuit/pathgen/BezierPath.java

```diff
@@ -0,0 +1,214 @@
+package io.github.frc5024.purepursuit.pathgen;
+
+import java.util.ArrayList;
+import ca.retrylife.ewmath.MathUtils;
+import edu.wpi.first.wpilibj.geometry.Translation2d;
+import edu.wpi.first.wpilibj.util.Units;
+
+/**
+ * This class is used to generate points along a bezier curves
+ */
+public class BezierPath extends Path{
+
+	// Maximum space allowed between points
+	private double maxSeperation = Units.inchesToMeters(4);
+
+	// The lookup table
+	private static ArrayList<int[]> binomialLookUpTable;
+
+	/**
+	 * 
+	 * @param wayPoints The 3 way points of a quadratic bezier curve
+	 */
+	public BezierPath(Translation2d[] wayPoints) {
+		this(wayPoints, new double[] { 1, 1, 1 }, Units.inchesToMeters(4));
```

Comment:
If you make the second argument an empty array, `new double[0]`, then you can rely on the other constructor to fill it in for you (down below)